app/view_objects/trainee_personal_details.rb

```diff
@@ -0,0 +1,32 @@
+class TraineePersonalDetails
```

Comment:
This looks like a decorator/presenter. Can we have a directory to reflect that? Maybe let's use `presenters` instead of `view_objects`

Reply:
view component?

Reply:
We already have view components set up in https://github.com/DFE-Digital/register-trainee-teacher-data/pull/7/files#diff-9c0af7b78005aba209897a91d821c36aR1. This object is a presenter to format model data into a specific way which is then passed to the component

Reply:
Cant you use view components at a presentational level too. (Component within a component).

Reply:
I think I have a bad taste with presenter/decorators. All i can think of is Draper being added, then a million methods being added to the presenter class. We can rename it as long as we don't go down that path

Reply:
i've renamed this to `presenters` now

Reply:
definitely don't need draper, we can just stick with lightweight/composable POROs